transcription disc scripts/moverestorations.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# microservice to move older transcription disc restorations into the restoration_old directory after a newer restoration is made using the mm/makederiv microservice
+
+# color codes for messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+_usage(){
+    echo -e ${GREEN}"\nThis script moves older transcription disc restorations into the restoration_old directory after a newer restoration is made using the mm/makederiv microservice.\n"${NC}
+}
+
+_usage
+
+# asks for a directory and checks it
+
+echo "Enter the directory path to begin:"
+read -r -p "> " user_directory
+
+if [[ ! -d "$user_directory" || -z "$(ls -A "$user_directory")" ]]; then
+    echo -e ${RED}"Directory not found or empty: $user_directory"${NC}
+    exit 1
+fi
+
+echo -e ${GREEN}"Selected directory: $user_directory"${NC}
+
+# is there a restoration directory? if no, exit
+restoration_directory="$user_directory/objects/restoration"
+
+if [ ! -d "$restoration_directory" ]; then
+    echo -e ${RED}"There is no 'restoration' directory in the objects directory. Aborting..."${NC}
+    exit 1
+fi
+
+# does it have wav files? if no, exit
+wav_files=$(find "$restoration_directory" -maxdepth 1 -type f -name "*.wav")
+
+if [ -z "$wav_files" ]; then
+    echo -e ${RED}"No WAV files found in the restoration directory. Aborting..."${NC}
+    exit 1
+fi
+
+# create restoration_old directory
+restoration_old_dir="$user_directory/objects/restoration/restoration_old"
+
+# append to readme.txt log file
+readme_file="$user_directory/metadata/oldrestoration_readme.txt"
+user=$(whoami)
+echo "Old transcription disc restoration file move log" > "$readme_file"
+echo "Initiated by: $user" >> "$readme_file"
+echo "==============================" >> "$readme_file"
+
+# identify old & new wav file
+for wav_file in $wav_files; do
```

Comment:
do you confirm this works for packages with multiple wav files in the restoration folder?

Reply:
yup. it considers multiple wav files with suffixes(?) like SDA255_1.wav and SDA255_01.wav.

attached is the readme that the script produces that logs the file move to the 'restoration_old' directory

[oldrestoration_readme.txt](https://github.com/mediamicroservices/mm/files/14299526/oldrestoration_readme.txt)